app/android/app/src/main/java/ca/bc/gov/BCWallet/MainApplication.java

```diff
@@ -4,6 +4,7 @@
 import android.content.Context;
 import com.facebook.react.PackageList;
 import com.facebook.react.ReactApplication;
+import com.oblador.vectoricons.VectorIconsPackage;
```

Comment:
what is this extra import for? it doesn't seem to be used

Reply:
That came from linking `react-native-vector-icons`. I can try removing it and see if it still works